src/testsuite/arith128_test_i64.c

```diff
@@ -393,102 +452,233 @@ test_vsrad (void)
 #undef __DEBUG_PRINT__
 
 int
-test_permdi (void)
+test_mrghld (void)
 {
   vui64_t i1, i2, e;
   vui64_t j;
   int rc = 0;
 
-  printf ("\ntest_permdi Vector permute doubleword immediate\n");
+  printf ("\ntest_merge Vector Merge High/Low Doubleword\n");
```

Comment:
s/test_merge/test_mrghld/
or, (globally) start using `__FUNCTION__` ?

Reply:
This is actually more complicated as 
printf ("__FUNCTION__  Vector Splat Doubleword Immediate\n");

will just print: "__FUNCTION__ Vector Splat Doubleword Immediate"

and: printf ("%s  Vector Splat Doubleword Immediate\n", __FUNCTION__);

generates more code (does not reduce to puts()). Not really an improvement

So deferring this for later.